src/main/java/cinema/Main.java

```diff
@@ -55,5 +58,16 @@ public static void main(String[] args) {
         } catch (AuthenticationException e) {
             e.printStackTrace();
         }
+        User user1 = new User();
+        user1.setEmail("sjfakljsflaks");
+        user1.setPassword("qwerty123");
+        UserService userService = (UserService) injector
+                .getInstance(UserService.class);
+        userService.add(user1);
+        ShoppingCartService shoppingCartService = (ShoppingCartService) injector
+                .getInstance(ShoppingCartService.class);
+        shoppingCartService.registerNewShoppingCart(user1);
```

Comment:
let's add shopping cart registration to authenticationService.register()

Reply:
done